openapi/azure.openai.deployment/openapi.yaml

```diff
@@ -0,0 +1,439 @@
+swagger: '2.0'
+info:
+  x-ballerina-display:
+        label: Azure OpenAI Deployment
+        iconPath: "icon.png"
+  title: Azure OpenAI API version 2022-12-01
+  description: APIs for managing deployments of OpenAI models.
+  x-ballerina-init-description: >
+      The connector initialization requires setting the API credentials.
+      Create an [Azure](https://azure.microsoft.com/en-us/features/azure-portal/) account, an [Azure OpenAI resource](https://learn.microsoft.com/en-us/azure/cognitive-services/openai/how-to/create-resource) and refer [this guide](https://learn.microsoft.com/en-us/azure/cognitive-services/openai/reference#authentication) to learn how to generate and use tokens
+  version: '2022-12-01'
+paths:
+  /deployments:
+    get:
+      tags:
```

Comment:
Add `x-ballerina-display` to all endpoints. Refer
https://github.com/hevayo/ballerina-openai-client/blob/fixes/client_openapi.yaml#L22-L23